lib/routes/account.js

```diff
@@ -442,15 +452,19 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
+        function sendVerifyLoginEmail() {
+          if (wantsKeys(request)) {
```

Comment:
FWIW, I agree that when we get better at differentiating "suspicious login attempt" from "normal login attempt" then it makes sense to send these emails regardless of relier; with the current approach of sending on every login attempt I think it would be a bit much.  But as always, am happy to have my mind changed!